src/test/common/router/routablePath.ts

```diff
@@ -65,4 +72,32 @@ describe('RoutablePath', () => {
       })
     })
   })
+
+  describe('isInValidParameterValue', () => {
```

Comment:
When you move it to outside class we will need to remove that test. Thats is alright, you testing this in other tests.

Reply:
done